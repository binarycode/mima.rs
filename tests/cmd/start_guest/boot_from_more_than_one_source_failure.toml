args = 'start-guest zero --boot-from-network --boot-from-cdrom'
status.code = 2
