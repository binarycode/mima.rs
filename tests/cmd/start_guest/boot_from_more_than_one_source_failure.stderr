error: the argument '--boot-from-network' cannot be used with '--boot-from-cdrom'

Usage: mima start-guest --boot-from-network <GUEST_ID>

For more information, try '--help'.
